proposals/multi-value/Overview.md

```diff
@@ -30,7 +30,7 @@
 
 * Inputs to blocks:
   - loop labels can have arguments
-  - can represent phis on backward edges
+  - can represent PHIs on backward edges
```

Comment:
"phi" is a greek letter, not an acronym, so this should remain lowercase.

Reply:
okie